src/components/Atoms/Checkbox/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import styled from "styled-components";
+import check from "../../../assets/svg/check.svg";
+import COLOR from "../../../variables/color";
+
+const Checkbox = (props) => {
```

Comment:
```suggestion
const Checkbox = ({onClick}) => {
```
こういう感じでJSの分割代入の機能を使って書くのが、どのようなPropsが存在するのかが分かりやすくなるのでおすすめです。

Reply:
了解です👌